build/toolsets/linux-x64.mak

```diff
@@ -10,6 +10,7 @@ DLL_SUFFIX        := .so
 DLL_PREFIX        := lib
 DLL_PATH          := LD_LIBRARY_PATH
 COMMON_CFLAGS     += -DLINUX -fPIC
+COMMON_CFLAGS     += -Wno-strict-aliasing
```

Comment:
Maybe it's better to disable strict aliasing instead? See discussion here https://stackoverflow.com/a/99010.

Reply:
Agree. Changed to `-fno-strict-aliasing`. Thank you for the link.